### PR TITLE
Feature iops 1945 spell checker corrections conflicts

### PR DESCRIPTION
**Pull request by KazeemHamzat**

### CODE REVIEW

File: codesystems/CodeSystem-England-ERSSpecialty.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
             <rank value="1" />
         </telecom>
     </contact>
-    <description value="Codesystem to desribe ERS Specialty" />
+    <description value="A Codesystem use to describe ERS Specialty" />
```
Comment (digitalhealthmartin) [inline codesystems/CodeSystem-England-ERSSpecialty.xml]: Codesystem -> CodeSystem

File: codesystems/CodeSystem-England-ERSSpecialty.xml [hunk 2]
```diff
@@ -241,8 +241,8 @@
         <display value="Rheumatology" />
     </concept>
     <concept>
-        <code value="CHILDRENS_ADOLESCENT_SERVICES" />
-        <display value="Childrens &amp; Adolescent Services" />
+        <code value="CHILDREN's_ADOLESCENT_SERVICES" />
```
Comment (digitalhealthmartin) [inline codesystems/CodeSystem-England-ERSSpecialty.xml]: Reverse change. Changing codes not in scope.

File: codesystems/CodeSystem-England-ERSSpecialty.xml [hunk 3]
```diff
@@ -241,8 +241,8 @@
...
+        <display value="Children's &amp; Adolescent Services" />
```
Comment (digitalhealthmartin) [inline codesystems/CodeSystem-England-ERSSpecialty.xml]: Reverse change. Changing codes not in scope.

File: codesystems/CodeSystem-England-HTTPErrorCode.xml [hunk 4]
```diff
@@ -42,23 +42,23 @@
         <definition value="The Server has given a 400 response to the Senders request." />
     </concept>
     <concept>
-        <code value="SEND_UNAUTHORIZED" />
-        <display value="401: The API deemed you unauthorized to make this request." />
+        <code value="SEND_UNAUTHORISED" />
```
Comment (digitalhealthmartin) [inline codesystems/CodeSystem-England-HTTPErrorCode.xml]: Reverse this code value changes and all code value changes.

File: codesystems/CodeSystem-England-MessageEventEPS.xml [hunk 5]
```diff
@@ -98,7 +98,7 @@
      <concept>
         <code value="encounter"/>
         <display value="Patient Encounter"/>
-        <definition value="Can be used for an admission/discharge report or consulations"/>
+        <definition value="Can be used for an admission/discharge report or consultations"/>
```
Comment (digitalhealthmartin) [inline codesystems/CodeSystem-England-MessageEventEPS.xml]: consultations -> consultation

File: codesystems/CodeSystem-England-HTTPErrorCode.xml (hunk 4 above)
Comment (digitalhealthmartin) [reply codesystems/CodeSystem-England-HTTPErrorCode.xml]: Also reverse display name changes

File: codesystems/CodeSystem-England-OrganisationRole.xml [hunk 6]
```diff
@@ -163,7 +163,7 @@
     </concept>
     <concept>
         <code value="215"></code>
-        <display value="HOSTS DATA MANAGMENT INTEGRATION CENTRE (DMIC)"></display>
+        <display value="HOSTS DATA MANAGEMENT INTEGRATION CENTRE (DMIC)"></display>
```
Comment (digitalhealthmartin) [inline codesystems/CodeSystem-England-OrganisationRole.xml]: Reverse display value changes.

File: codesystems/CodeSystem-England-PrescriptionType.xml [hunk 7]
```diff
@@ -227,7 +227,7 @@
 	</concept>
 	<concept>
 		<code value="0124"/>
-		<display value="Primary Care Prescriber – Dietician Supplementary prescriber"/>
+		<display value="Primary Care Prescriber – Dietitian Supplementary prescriber"/>
```
Comment (digitalhealthmartin) [inline codesystems/CodeSystem-England-PrescriptionType.xml]: Reverse changes

Comment (digitalhealthmartin) [reply codesystems/CodeSystem-England-PrescriptionType.xml]: For this and all codes and definition values

File: valuesets/ValueSet-England-DMPrescriptionType.xml [hunk 8]
```diff
@@ -51,7 +51,7 @@
         <contains>
             <system value="https://fhir.nhs.uk/CodeSystem/England-PrescriptionType"/>
             <code value="0924"/>
-            <display value="Private prescribing - Dietician"/>
+            <display value="Private prescribing - Dietitian"/>
```
Comment (digitalhealthmartin) [inline valuesets/ValueSet-England-DMPrescriptionType.xml]: Reverse change for this and other codes/display values

File: valuesets/ValueSet-England-OrganisationRole.xml [hunk 9]
```diff
@@ -183,7 +183,7 @@
             </concept>
             <concept>
                 <code value="215" />
-                <display value="HOSTS DATA MANAGMENT INTEGRATION CENTRE (DMIC)" />
+                <display value="HOSTS DATA MANAGEMENT INTEGRATION CENTRE (DMIC)" />
```
Comment (digitalhealthmartin) [inline valuesets/ValueSet-England-OrganisationRole.xml]: Reverse change

**Review by digitalhealthmartin — CHANGES_REQUESTED** [review]

File: codesystems/CodeSystem-England-HTTPErrorCode.xml [hunk 10]
```diff
@@ -98,17 +98,17 @@
     </concept>
     <concept>
         <code value="SEND_METHOD_NOT_ALLOWED" />
-        <display value="405: This API doesnt allow this method." />
+        <display value="405: This API doesn't allow this method." />
```
Comment (digitalhealthmartin) [inline codesystems/CodeSystem-England-HTTPErrorCode.xml]: This needs reverting

File: codesystems/CodeSystem-England-HTTPErrorCode.xml [hunk 11]
```diff
@@ -98,17 +98,17 @@
...
         <definition value="The API has given a 405 response to the Senders request." />
     </concept>
     <concept>
         <code value="REC_METHOD_NOT_ALLOWED" />
-        <display value="405: The Receiver doesnt allow this method." />
+        <display value="405: The Receiver doesn't allow this method." />
```
Comment (digitalhealthmartin) [inline codesystems/CodeSystem-England-HTTPErrorCode.xml]: This needs reverting

File: codesystems/CodeSystem-England-HTTPErrorCode.xml [hunk 12]
```diff
@@ -98,17 +98,17 @@
...
         <definition value="The Receiver has given a 405 response to the Senders request." />
     </concept>
     <concept>
         <code value="PROXY_METHOD_NOT_ALLOWED" />
-        <display value="405: The Proxy doesnt allow this method." />
+        <display value="405: The Proxy doesn't allow this method." />
```
Comment (digitalhealthmartin) [inline codesystems/CodeSystem-England-HTTPErrorCode.xml]: This needs reverting

File: codesystems/CodeSystem-England-HTTPErrorCode.xml [hunk 13]
```diff
@@ -134,7 +134,7 @@
     <concept>
         <code value="REC_TIMEOUT" />
         <display value="408: The request timed out to the receiver." />
-        <definition value="The Receiver has given a 408 resonse to the Senders request" />
+        <definition value="The Receiver has given a 408 response to the Senders request" />
```
Comment (digitalhealthmartin) [inline codesystems/CodeSystem-England-HTTPErrorCode.xml]: This needs reverting

File: codesystems/CodeSystem-England-MessageEventEPS.xml [hunk 14]
```diff
@@ -74,7 +74,7 @@
     <concept>
         <code value="dispense-notification-update"/>
         <display value="Dispense Notification Update"/>
-        <definition value="This message is used by the Dispensing System to notify the EPS System of an amendment to a previousy submitted dispense-notification"/>
+        <definition value="This message is used by the Dispensing System to notify the EPS System of an amendment to a previously submitted dispense-notification"/>
```
Comment (digitalhealthmartin) [inline codesystems/CodeSystem-England-MessageEventEPS.xml]: This needs reverting

File: codesystems/CodeSystem-England-MessageEventEPS.xml [hunk 15]
```diff
@@ -98,7 +98,7 @@
      <concept>
         <code value="encounter"/>
         <display value="Patient Encounter"/>
-        <definition value="Can be used for an admission/discharge report or consulations"/>
+        <definition value="Can be used for an admission/discharge report or consultation"/>
```
Comment (digitalhealthmartin) [inline codesystems/CodeSystem-England-MessageEventEPS.xml]: This needs reverting

**Review by digitalhealthmartin — CHANGES_REQUESTED** [review]

**Review by SheenaSharma — APPROVED** [review]

**Review by digitalhealthmartin — APPROVED** [review]

**Pull request merged**